### PR TITLE
small changes fro flutterv3

### PR DESCRIPTION
**Pull request by HeyitSaif**
changes required to upgrade for v3

### CODE REVIEW

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/NativeViewFactory.kt [hunk 1]
```diff
@@ -8,8 +8,8 @@ import io.flutter.plugin.platform.PlatformView
 import io.flutter.plugin.platform.PlatformViewFactory
 
 class NativeViewFactory(val activity: Activity) : PlatformViewFactory(StandardMessageCodec.INSTANCE) {
-    override fun create(context: Context, viewId: Int, args: Any?): PlatformView {
+    override fun create(context: Context?, viewId: Int, args: Any?): PlatformView {
         val creationParams = args as Map<String?, Any?>?
-        return NativeView(activity, context, viewId, creationParams)
+        return NativeView(activity, context!!, viewId, creationParams)
```
Comment (geiszla) [inline jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/NativeViewFactory.kt]: Don't ignore the nullability of the type here. Change the `NativeView` instead to accept a nullable type.

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt [hunk 2]
```diff
@@ -18,7 +19,7 @@ import org.jitsi.meet.sdk.JitsiMeetConferenceOptions
 /**
  * Activity extending JitsiMeetActivity in order to override the conference events
  */
-class JitsiMeetPluginActivity : JitsiMeetActivity() {
+class JitsiMeetPluginActivity<Activity> : JitsiMeetActivity() {
```
Comment (geiszla) [inline jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt]: I don't quite understand this. Won't this just add a new generic variable? Do we use that variable anywhere?

**Review by geiszla — CHANGES_REQUESTED** [review]

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt (hunk 2 above)
Comment (HeyitSaif) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt]: i don't understand what do you mean here. it just a return type here. v3 forces you to put retun type.

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/NativeViewFactory.kt (hunk 1 above)
Comment (HeyitSaif) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/NativeViewFactory.kt]: done!

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt (hunk 2 above)
Comment (geiszla) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt]: How does a class have return type? Isn't that a [generic type variable](https://kotlinlang.org/docs/generics.html)?

Comment (HeyitSaif) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt]: i mean it specifies the type of the class itself. it doesn't have the need to be generic. the class is extendeing formt he jitsimeetactivity which is extened further from the activity android. so for that we need to explicitly defined it's type otherwise it will not compile
Generic doesn't work

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt [hunk 3]
```diff
@@ -94,7 +94,6 @@ public class JitsiMeetPlugin() : FlutterPlugin, MethodCallHandler, ActivityAware
 
         pluginBinding = flutterPluginBinding;
 
-        flutterPluginBinding
```
Comment (geiszla) [inline jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt:97]: I just build the app and ran it successfully and I didn't need to change this. Why is this necessary?

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt (hunk 2 above)
Comment (geiszla) [inline jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt]: I also did not add this and the app built successfully. Can you check if this is needed?

**Review by geiszla — CHANGES_REQUESTED** [review]

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt (hunk 3 above)
Comment (HeyitSaif) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt:97]: this looks odd. it shouldn't be ther ein the first place as it is useless that's why i remove that

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt (hunk 2 above)
Comment (HeyitSaif) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt]: that's very strange if that works for you. i was getting this error on compile time and after some research i found the cause

Comment (HeyitSaif) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt]: btw this is not a crucial change in itself as well. it's just enforcing the type for the class. so it creates no harm in there.

File: jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt (hunk 3 above)
Comment (HeyitSaif) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt:97]: btw what the point of putting it there. it looks like a type in the code and should be removed.

Comment (geiszla) [reply jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPlugin.kt:97]: You're right, we can remove this.

**Review by geiszla — APPROVED** [review]
